proto/cosmos/base/reflection/v1beta1/reflection.proto

```diff
@@ -17,7 +17,7 @@ service ReflectionService {
   // interface.
   rpc ListImplementations(ListImplementationsRequest) returns (ListImplementationsResponse) {
     option (google.api.http).get = "/cosmos/base/reflection/v1beta1/interfaces/"
-        "{interface_name}/implementations";
+                                   "{interface_name}/implementations";
```

Comment:
the auto spacing seems a bit off here

Reply:
I think this is to line it up w/ line 19?

Reply:
It's auto formatted, we shouldn't change it.